fact_extractor/plugins/unpacking/squashFS/code/squash_fs.py

```diff
@@ -30,7 +30,20 @@ def unpack_function(file_path, tmp_dir):
     '''
     unpack_result = {}
     for unpacker, parameter in SQUASH_UNPACKER:
-        output = execute_shell_command(f'fakeroot {unpacker} {parameter} -d {tmp_dir}/fact_extracted {file_path}')
+        output = execute_shell_command(
+            " ".join(
+                [
+                    "fakeroot",
+                    str(unpacker),
+                    parameter,
+                    "-dest",
+                    str(tmp_dir),
+                    # We need to force here since "-dest" does not allow existing directories
+                    "-force",
+                    str(file_path),
+                ],
+            )
+        )
```

Comment:
I find this really hard to read. Why not simply put the comment on the line above? It's still obvious what it refers to IMHO

```suggestion
        # We need to force here since "-dest" does not allow existing directories
        output = execute_shell_command(f'fakeroot {unpacker} {parameter} -dest {tmp_dir} -force {file_path}')
```

Reply:
Done.